academia/lectures/Lecture - The Art of Not Following the Market/How To - Beta Hedging.ipynb

```diff
@@ -5,6 +5,16 @@
    "metadata": {},
    "source": [
     "#Beta Hedging\n",
+    "By David Edwards\n",
```

Comment:
For the notebooks that have companion backtests, we should have

Companion algorithms by David Edwards.